recipe/meta.yaml

```diff
@@ -25,17 +27,24 @@ test:
   requires:
     - flake8
     - pytest
+  commands:
+    - pytest -v --pyargs pyct
+    - pyct --help
+    - pyct --version
+    - pyct report --help
+    - pyct report pyct python
 
 about:
-  home: http://github.com/pyviz/pyct
+  home: http://github.com/pyviz-dev/pyct
   license: BSD-3-Clause
   license_family: BSD
   # in pyct-core
   # license_file: LICENSE.txt
```

Comment:
```suggestion
  license_file: LICENSE.txt
```

Reply:
(The PR is complaining about no license file; not sure what the comment "in pyct-core" means...

Reply:
I'm afraid I don't know. The recipe is different from the one I originally contributed, and I lost track of the changes.